### ISSUE TITLE
tp Week 5 regarding Travis-ci.com vs Travis-ci.org

### ISSUE THREAD

**Issue by khsc96** [nus-cs2103-AY1920S2/forum#35]
So my team and I have found out that there's two different domain for travis-ci (.org / .com)

We have found answers on these website:
https://devops.stackexchange.com/a/4305
https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps

Apparently, .com domain is the latest version for travis-ci and host both private and public repos. 
We are wondering do we still stick to .org, as my team have set both up and can't seem to delete either of them.

**Another issue we faced**
While exploring the request tab under my Repositories on travis ci (.com version), we have encountered this issue where gh-pages was not found. However, our build is able to pass and we are able to view our README hosted on Github Pages. Is it an issue or something that we have to fix?
Below is the attached image of how it looks like.

![image](https://user-images.githubusercontent.com/50527804/74015367-70ed2580-49cb-11ea-98b7-4cb970d6714a.png)

Thank you!

**Comment by okkhoy** [comment]
It should not matter as long as you can get auto-publishing to work and generate the UG/DG.

The instructions given in the Travis guide pertain to the `.org` (older) domain. 

Can you elaborate on the gh-pages issue? I am not sure I understood your question.

**Comment by khsc96** [comment]
Okay, so just to confirm the travis set up is completed as long as I see the travis bot automatically rebuilds the project whenever there's a new commit to the master branch and a commit appears under the 'gh-pages' branch is that it?

As for the gh-pages issue what I meant was if it is normal to get that image attached on my first post found at travis-ci webpage. 
How to get to that particular page is through my repository at travis-ci.org > more options >requests.

**Comment by okkhoy** [comment]
> Okay, so just to confirm the travis set up is completed as long as I see the travis bot automatically rebuilds the project whenever there's a new commit to the master branch and a commit appears under the 'gh-pages' branch is that it?

Yes.

Can I check if you did this: Setup a dummy account that has admin rights on your repo. For this dummy account, you need to generate the GitHub token and add the token to Travis.

The job rejected (cross mark with gh-pages) could be because the account doesn't have push access to your repo.

**Comment by khsc96** [comment]
Yes I did the setup of a dummy account that has admin rights to my repo already.

Oh alright, i'll go check the push access. Thanks!

**Comment by harrychengly** [comment]
Hi @khsc96, did you manage to solve this issue? My dummy account has admin rights to the repo already but the issue where gh-pages was not found still persist. 

What did you do to fix it?

**Comment by okkhoy** [comment]
Something that struck me just now: Have you enabled GitHub pages from gh-pages branch?
![image](https://user-images.githubusercontent.com/1023494/75623535-cd67ed00-5be5-11ea-89c6-3a4612a156eb.png)

**Comment by harrychengly** [comment]
Yes, I did. If not mistaken, it is shown like the attached photo right? 

![Screenshot 2020-03-01 at 5 58 02 PM](https://user-images.githubusercontent.com/28550646/75623603-86c6c280-5be6-11ea-85d0-d3f9c55b9a36.png)

**Comment by okkhoy** [comment]
Seems like your latest build passed: https://www.travis-ci.org/AY1920S2-CS2103-W14-4/main/builds

Am I looking at the correct page?

**Comment by harrychengly** [comment]
Yes, however when clicking on "More Options" -> "Requests", you will be able to see the issue regarding the gh-pages. 

![Screenshot 2020-03-01 at 6 10 48 PM](https://user-images.githubusercontent.com/28550646/75623743-289adf00-5be8-11ea-82d7-d3b70793765b.png)

Should I be worried about this?

**Comment by okkhoy** [comment]
Try this: pull your gh-pages branch as well, make changes to some doc files to initiate a doc build (this is as per normal; nothing to be done to gh-pages apart from pull) push the changes to your repo. Let me know what happens.

**Comment by chrisjwelly** [comment]
> 
> 
> It should not matter as long as you can get auto-publishing to work and generate the UG/DG.
> 
> The instructions given in the Travis guide pertain to the `.org` (older) domain.
> 
> Can you elaborate on the gh-pages issue? I am not sure I understood your question.

Hello! During Week 5, my team saw this post and we had set up Travis in the `.com` domain instead of `.org`.

It is rather late, but we just realised that this has caused a "Build Unknown" in the [TP showcase page](https://nus-cs2103-ay1920s2.github.io/website/admin/projectList.html#cs2103t-f10) as shown below:
![image](https://user-images.githubusercontent.com/39089453/78472904-0b20de00-776f-11ea-9e98-6c9587670756.png)

The badge is visible in our [team page](https://ay1920s2-cs2103t-f10-1.github.io/main/) and Travis has been running tests and auto-deployed our UG/DG. 

Is there any action that needs to be taken with regards to "Build Unknown" in TP showcase page?

**Comment by okkhoy** [comment]
@chrisjwelly Aah... that is generated using the script we have. So it checks for .org only.
Guess it doesn't matter. You can update the badge link on your repo to reflect the correct status.
